src/python/ensembl/io/genomio/assembly/status.py

```diff
@@ -0,0 +1,473 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Record the assembly status for a set of INSDC accessions using ncbi 'datasets' tool"""
+
+__all__ = [
+    "examine_parameterization",
+    "resolve_query_type",
+    "fetch_asm_accn",
+    "datasets_asm_reports",
+    "extract_assembly_metadata",
+    "generate_report_tsv",
+]
+
+import csv
+import json
+import os
+from os import PathLike, getcwd
+from pathlib import Path
+import re
+import logging
+import sys
+from typing import Dict, Tuple, Union
+
+from spython.main import Client
+
+from ensembl.io.genomio.utils.json_utils import print_json
+from ensembl.database import DBConnection as dbc
```

Comment:
```suggestion
from ensembl.io.genomio.database.dbconnection_lite import DBConnectionLite as dbc
```
After testing with lots of cores, I recommend using this wrapper which allows quicker access to the meta table (`DBConnection` proper loads the whole schema from the database, which takes a long time with a lot of genomes).

Reply:
Thanks Matthieu I shall implement this change for sure !! Did you notice a marked difference in runtime out of interest ?

Reply:
Done ! On the face of things it does appear to be a good bit quicker, nice one